src/cmd/ship.go

```diff
@@ -16,10 +16,11 @@ import (
 )
 
 type shipConfig struct {
-	BranchToShip  string
-	InitialBranch string
+	BranchToShip  string // the name of the branch to ship
+	InitialBranch string // the name of the branch that was checked out when running this command
 }
 
+// optional commit message provided via the command line
```

Comment:
Again, not sure what value this has. Understand this can be useful to give context faster but would like to do this in the code when possible. Appears we only have a few other flags in use. Could we rename this to make it more clear? Definitely prefer comments about why with code self documenting what.